README.md

```diff
@@ -38,7 +38,17 @@ We assume you have the following installed:
   * Erlang
   * Elixir
   * Node
-  * PostgreSQL
+
+#### PostgreSQL
+
+You may install PostgreSQL directly on your development computer or use a `docker` container
+using the [docker-compose](./docker-compose.yml) setup with a simple command:
+
+```shell
+> docker-compose up -d
+```
+
+The `DATABASE_URL` configuration shown below assume you are using the `docker` container.
 
 ### Installing
```

Comment:
Did you have a chance to run through the [installation instructions](https://github.com/gcauchon/montreal-elixir-website/tree/feature/development-config-as-env#installing). I started doing this and the following instruction failed:

```
$ mix do ecto.create, ecto.migrate
** (KeyError) key :database not found in: [otp_app: :montreal_elixir, repo: MontrealElixir.Repo, timeout: 15000, pool_timeout: 5000, adapter: Ecto.Adapters.Postgres, pool_size: 10]
    (elixir) lib/keyword.ex:377: Keyword.fetch!/2
    lib/ecto/adapters/postgres.ex:110: Ecto.Adapters.Postgres.storage_up/1
    lib/mix/tasks/ecto.create.ex:40: anonymous fn/3 in Mix.Tasks.Ecto.Create.run/1
    (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:737: Enum.each/2
    (mix) lib/mix/task.ex:314: Mix.Task.run_task/3
    (elixir) lib/enum.ex:737: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:737: Enum.each/2
    (mix) lib/mix/task.ex:314: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:80: Mix.CLI.run_task/2
    (elixir) lib/code.ex:677: Code.require_file/2
```

The issue here is that we are running this before we configure the database. It would be good to verify that the installation instructions work before we merge this in.

Reply:
I switched the `Configuration` and `Installing` section in the doc.